openwisp_monitoring/check/settings.py

```diff
@@ -5,9 +5,13 @@
     (
         ('openwisp_monitoring.check.classes.Ping', 'Ping'),
         ('openwisp_monitoring.check.classes.ConfigApplied', 'Configuration Applied'),
+        ('openwisp_monitoring.check.classes.Iperf', 'Iperf'),
     ),
 )
 AUTO_PING = get_settings_value('AUTO_PING', True)
 AUTO_CONFIG_CHECK = get_settings_value('AUTO_DEVICE_CONFIG_CHECK', True)
 MANAGEMENT_IP_ONLY = get_settings_value('MANAGEMENT_IP_ONLY', True)
 PING_CHECK_CONFIG = get_settings_value('PING_CHECK_CONFIG', {})
+AUTO_IPERF = get_settings_value('AUTO_IPERF', False)
+IPERF_SERVERS = get_settings_value('IPERF_SERVERS', {})
+IPERF_CHECK_CONFIG = get_settings_value('IPERF_CHECK_CONFIG', {})
```

Comment:
`OPENWISP_MONITORING_CHECKS` should go here and be inferred from `CHECK_CLASSES` automatically, eg:

```python
CHECKS_LIST = list(dict(CHECK_CLASSES).keys())
```

Reply:
- Added `CHECKS_LIST` in `checks/settings.py` and remained same celery beat config for **ping** and **config applied** check. 
- Added instructions in  `How to configure iperf check` to configure celery beat for **iperf** check.